cli/src/main/java/ai/wanaku/cli/main/support/FileHelper.java

```diff
@@ -0,0 +1,28 @@
+package ai.wanaku.cli.main.support;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class FileHelper {
+
+    private FileHelper() {}
+
+    public static boolean canWriteToDirectory(Path dir) {
+        if (dir == null) {
+            return false;
+        }
+
+        try {
+            Path tempFile = Files.createTempFile(dir, "test", ".tmp");
+            Files.delete(tempFile);
+            return true;
+        } catch (IOException e) {
```

Comment:
Maybe log here.

Reply:
This is just an helper method to check if a directory is writable, does it make sense to log something?

Reply:
Ah, got it. No worries, then.